blog/optimizations-to-make-to-your-docs-for-llms.md

```diff
@@ -0,0 +1,285 @@
+
+The blue sections represent the developer experience we should strive for: clear questions get clear answers, code examples actually work, and developers can focus on building rather than debugging documentation problems.
+
+```mermaid
+sequenceDiagram
+    participant Dev as Developer
+    participant AI as AI Assistant
+    participant Good as Optimized Docs
+    participant Bad as Unoptimized Docs
+    
+    Note over Dev,Bad: Poor Documentation Experience
+    
+    rect rgb(255, 200, 150)
+        Note right of Dev: Authentication Request
+        Dev->>+AI: "How to authenticate?"
+        AI->>+Bad: Search inconsistent docs
+        Bad-->>-AI: Mixed terminology, unclear structure
+        AI-->>-Dev: Confusing response with multiple terms
+    end
+    
+    rect rgb(255, 180, 120)
+        Note right of Dev: Code Example Request
+        Dev->>+AI: "Show me code example"
+        AI->>+Bad: Retrieve unformatted code
+        Bad-->>-AI: Inline code without proper formatting
+        AI-->>-Dev: Broken/malformed code example
+    end
+    
+    Note over Dev,Bad: Developer gets frustrated, creates support ticket
+    
+    Note over Dev,Good: Optimized Documentation Experience
+    
+    rect rgb(173, 216, 230)
+        Note right of Dev: Authentication Request
+        Dev->>+AI: "How to authenticate?"
+        AI->>+Good: Search well-structured docs
+        Good-->>-AI: Consistent terminology, clear hierarchy
+        AI-->>-Dev: Clear, accurate authentication guide
+    end
+    
+    rect rgb(135, 206, 250)
+        Note right of Dev: Code Example Request
+        Dev->>+AI: "Show me code example"
+        AI->>+Good: Retrieve properly formatted code
+        Good-->>-AI: Fenced code blocks with language tags
+        AI-->>-Dev: Correct, runnable code example
+    end
+    
+    Note over Dev,Good: Developer succeeds quickly
+```
+
+The good news? Improving how LLMs consume your docs isn't particularly complex. It just requires a little rethinking about how you structure and format your documentation with AI consumption in mind.
+
+
+## Common challenges with API docs and LLMs
+
+AI models don't read documentation like humans do. They can't look at your beautiful layout, understand visual hierarchy, or fill in context gaps the way a developer can. Instead, they process text in a very specific way that documentarians may not have considered in the past.
+
+### Mixed topics confuse AI chunking
+
+AI systems organize information into chunks, usually paragraph-sized sections. When you mix different topics in the same section (like putting rate limiting info in your authentication section), the AI can't separate the concepts. So when someone asks about authentication, they get rate limiting details mixed in, making the answer less useful.
+
+Some tool can help - Redocly, for example, automates the creation of your `llms.txt` file with a lightweight Markdown index that may help AI assistants navigate your docs more intelligently.
```

Comment:
I would remove this -- as least doesn't fit within this heading (not related to chunking).